x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm@settings.yaml

```diff
@@ -6,4 +6,4 @@ _meta:
 template:
   settings:
     codec: best_compression
```

Comment:
I would also update the codec setting here to `index.codec`.

Reply:
Ah thanks 😄

Reply:
Is it worth creating the nesting then?